lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -218,79 +226,17 @@ <h3 class="course-title">
               </button>
               <div class="actions-dropdown" id="actions-dropdown-${dashboard_index}" tabindex="-1">
```

Comment:
Why can't we move the content here into a new template, and create a model with an iframe?